changelog/install-dmd-dc.dd

```diff
@@ -0,0 +1,27 @@
+`~/dlang/install.sh` now supports `--dmd` and `--dc`
+
+`--dmd` and `--dc` has been added as a new option to the install script.
+These flags work analogous to the existing `--activate` flag and
+print information about the installed compiler binary:
+
+$(UL
+    $(LI `--dmd` returns the path of a specific compiler executable (DMD-like interface))
```

Comment:
What will this be if the compiler does not expose a DMD-like interface?

Reply:
I don't think SDC will grow into a full compiler any time soon and even if, for a seamless experience it most likely will implement the DMD interface (IIRC it already does so).
The same applies for other compilers and if there's ever a case in a far far far distant future where we would want to include a new compiler, but are too lazy to wrap the DMD interface around it, `fatal` error is the Unix way.
Also as mentioned this is the changelog entry - not the documentation, so imho there's no need to go into details here.

Reply:
> Also as mentioned this is the changelog entry - not the documentation, so imho there's no need to go into details here.

I didn't ask for the changelog to be altered.  I asked what the result will be if the compiler does not expose a DMD-like interface.

Reply:
This can't happen as `install.sh` only supports GDC, DMD and SDC. If there's ever a new compiler which would be added to `install.sh`, it would fail, I presume.